src/terminal/input/terminalInput.cpp

```diff
@@ -231,29 +268,44 @@ void TerminalInput::ChangeCursorKeysMode(const bool applicationMode) noexcept
 }
 
 static const std::basic_string_view<TermKeyMap> _getKeyMapping(const KeyEvent& keyEvent,
+                                                               const bool ansiMode,

```

Comment:
nit: add this to the doc comment

Reply:
Sorry, what doc comment are you referring to? Did you want me to add one for this function, or is there an existing comment somewhere that I should update?

Reply:
Oh whoops, didn't see there wasn't one. You can just ignore it then 😅